src/data/battle_frontier/trainer_hill.h

```diff
@@ -382,18 +383,18 @@ static const struct TrHillFloor gUnknown_0862609C_floors[] = {
 
 static const struct TrHillTag sDataTagNormal =
 {
-    .unkField_0 = 8,
+    .numTrainers = NUM_TRAINER_HILL_TRAINERS,
     .unused1 = 2,
-    .numFloors = 4,
+    .numFloors = NUM_TRAINER_HILL_FLOORS,
     .checksum = 0x00051E05
 };
 
-static const struct TrHillFloor sDataTagNormal_floors[] =
+static const struct TrHillFloor sDataTagNormal_Floors[] =
 {
     [0] =
     {
-        .unk0 = 0x11,
-        .unk1 = 0x12,
+        .trainerNum1 = 17,
```

Comment:
Are these numbers supposed to be opponent constants, etc.?

Reply:
I don't think so. The actual `TRAINER_` value for the trainer on each floor is just 1 and 2 each time (see the use of `gSaveBlock2Ptr->frontier.trainerIds` in `trainer_hill.c`. As far as I can tell those are some kind of identifier for each trainer used to determine what prize to give. If you mean the use of the name `trainerNum` is misleading here then I'm happy to rename it.

Reply:
it's fine for now

Reply:
Yeah there's clearly some funky things going on with trainer hill. I think documenting more of E-Reader might clear it up.